examples/network/hierarchy_network_demo.py

```diff
@@ -45,7 +45,7 @@
 
 # Seed used for random number generation
 _SEED = 2045
-_INPUT_FILE_PATH = "../prediction/data/extra/hotgym/rec-center-hourly.csv"
+_INPUT_FILE_PATH = "extra/hotgym/rec-center-hourly.csv"
```

Comment:
I think it would be better to use the `resource_filename` here, otherwise you define the relative path in one place and the package it is relative to later in the code